SETUP_ARCH.md

```diff
@@ -0,0 +1,23 @@
+# Snowdrift setup instructions for Arch Linux
+
+Arch has the benefit of being very well-documented, so this document is
+very short!
+
+1.  [Install Haskell][1]. While you are at it, make sure you have the
+    `base-devel` group installed.
+2.  [Install PostgreSQL][2]. You don't need to go through setting up a
+    user or anything like that. Snowdrift will do that for you.
+3.  Run these commands:
+
+        git clone https://git.gnu.io/snowdrift/snowdrift.git
+        cd snowdrift
+        ln -s cabal.config.7.10 cabal.config
+        cabal sandbox init
+        cabal install -j -fdev --enable-tests
+        sdm init
+        Snowdrift Development
+
+The site should now be running on <http://localhost:3000>.
+
+[1]: https://wiki.archlinux.org/index.php/Haskell
+[2]: https://wiki.archlinux.org/index.php/Postgresql
```

Comment:
Having looked at those links now, they seem pretty thorough and clunky. They go through explaining all sorts of things and providing tons of options and things to make sense of and do or not do.

Would it not be easier to just select out the most concise install commands, like with the Debian guide, and include them here?

To be clear, we *already* mention Arch in the GUIDE by linking to https://www.haskell.org/downloads/linux
Importantly, that link has *MUCH* shorter and clearer instructions. I think we should use that or otherwise have instructions here in this file that are of that ilk. If we really want, we can link to these much longer things somewhere…